ramls/item.json

```diff
@@ -296,6 +296,45 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
+      "type": "object",
+      "readonly": true,
+      "additionalProperties": false,
+      "properties": {
+        "date": {
+          "type": "string",
+          "description": "Date and time of the last check in of the item."
+        },
+        "servicePointName": {
+          "type": "string",
+          "description": "Service point being used by user when item was scanned in Check in app."
+        },
+        "source": {
+          "type": "object",
+          "readonly": true,
+          "additionalProperties": false,
+          "description": "Lastname, first name middle name (with link to user profile) of user who scanned the item",
+          "properties": {
+            "id": {
```

Comment:
The description suggests this is the `user ID` of the staff member or patron who performed the check in, could the name of this property be made more specific to reflect what kind of record this is intended to reference and what role they were performing.

Reply:
As this is intended to be a UUID, please add a format to constrain the contents of this property to a UUID

Reply:
Done.